recipes/cec1/e009_sheffield/test.py

```diff
@@ -25,15 +25,21 @@ def run(cfg: DictConfig) -> None:
 
     down_sample = up_sample = None
     if cfg.downsample_factor != 1:
+        if torchaudio.__version__[0] == "0":
```

Comment:
I'd rather we just drop the support for the v0 torch audio. By the time we release v0.4 torch 2.0 and the new torchaudio will have been available a long time. The dependencies already have torch>=2